### ISSUE TITLE
Write out Geth version on start

### ISSUE THREAD

**Issue by SteffenIvanhoe** [ethereum/go-ethereum#2609]
#### System information

Geth version: every
OS & Version: Windows/Linux/OSX
Commit hash : (if `develop`)
#### Expected behaviour

Write out the Geth version to console window on start as first line.
#### Actual behaviour

Version not identifiable.
#### Steps to reproduce the behaviour

Just start Geth in the console window.
#### Backtrace

```
[backtrace]
```

**Comment by bas-vk** [comment]
@fjl, maybe I'm misunderstanding @SteffenIvanhoe but this is a bug report and not a request for a new feature? The console should print something like this:

```
instance: Geth/v1.5.0-unstable-d9bb8179/linux/go1.6.2
```

I tried it on the Windows command prompt and powershell and in both cases it would print out the version. SteffenIvanhoe could you maybe explain a bit more what is the problem is?

**Comment by SteffenIvanhoe** [comment]
Sorry, this is a request and no bug report. If I start `geth.exe` on Windows it does not show a line containing a version id. I for example get 

```
I0525 14:11:16.653102 ethdb/database.go:82] Alloted 128MB cache and 1024 file handles to C:\Users\Steffen\AppData\Roaming\Ethereum\chaindata
I0525 14:11:17.137917 ethdb/database.go:169] closed db:C:\Users\Steffen\AppData\Roaming\Ethereum\chaindata
I0525 14:11:17.144928 ethdb/database.go:82] Alloted 128MB cache and 1024 file handles to C:\Users\Steffen\AppData\Roaming\Ethereum\chaindata
I0525 14:11:17.310455 ethdb/database.go:82] Alloted 16MB cache and 16 file handles to C:\Users\Steffen\AppData\Roaming\Ethereum\dapp
I0525 14:11:17.317500 eth/backend.go:172] Protocol Versions: [63 62 61], Network Id: 1
I0525 14:11:17.318503 eth/backend.go:201] Blockchain DB Version: 3
I0525 14:11:17.320504 core/blockchain.go:206] Last header: #1580919 [53937543.] TD=21989251773569414589
I0525 14:11:17.321505 core/blockchain.go:207] Last block: #1580919 [53937543.] TD=21989251773569414589
I0525 14:11:17.322506 core/blockchain.go:208] Fast block: #1580919 [53937543.] TD=21989251773569414589
I0525 14:11:17.325509 p2p/server.go:311] Starting Server
I0525 14:11:19.002824 p2p/discover/udp.go:217] Listening, enode://f43657f52ec483b6c2b58438e0daf125ecff64dd6fb7dc97f3c5255658860b46a06e11abc5a9dc88d26086dba40a500de1bd6000cfa385433b45edde84d70fed@[::]:30303
I0525 14:11:19.009830 p2p/server.go:554] Listening on [::]:30303
I0525 14:11:19.009830 node/node.go:298] IPC endpoint opened: \\.\pipe\geth.ipc
```

I can find out the version with `geth version` but would prefer the program to start with an output line which identifies the version. This is especially helpful when working with and maintaining multiple `geth` versions. I'm used to seeing that self-identification behavior with many other command line tools and would like to see it in `geth` too. Is also helpful for protocol outputs redirected into files.

**Comment by fjl** [comment]
I agree that this would be nice to have, hence the feature label.

**Comment by fjl** [comment]
I agree that this would be nice to have, hence the feature label.

**Comment by fjl** [comment]
I agree that this would be nice to have, hence the feature label.

**Comment by fjl** [comment]
Fix released in 1.5.0.
